cmd/queue/main.go

```diff
@@ -305,14 +296,20 @@ func main() {
 		os.Exit(0)
 	}
 
-	logger, _ = logging.NewLogger(os.Getenv("SERVING_LOGGING_CONFIG"), os.Getenv("SERVING_LOGGING_LEVEL"))
+	var env config
+	if err := envconfig.Process("", &env); err != nil {
+		fmt.Fprintln(os.Stderr, err)
```

Comment:
Is the error self-descriptive here?

Reply:
yes.
Verified by not passing ```required``` key

Error:
**required key ```USER_PORT``` missing value**